Protocol/Tests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag.cs

```diff
@@ -193,7 +199,7 @@ public void CheckUsedFeatures()
 
             if (subResults.Count > 0)
             {
-                IValidationResult minVersionTooLow = Error.MinVersionTooLow(test, null, GetPositionNode(), versionValue, expectedVersion.ToString());
+                IValidationResult minVersionTooLow = Error.MinVersionTooLow(test, null, GetPositionNode(), tag?.RawValue, expectedVersion.ToString());
```

Comment:
This changed from tag?.value to tag?.rawvalue. Can you verify that was the intention?

Reply:
Yes as the RawValue is the actual value in the tag itself. This is done to make sure that the user can't be confused by seeing a different value (trimmed, parsed/converted) than what is specified in the tag.